File: app/src/main/java/com/dlka/lastfmnewtracknotifier/Image.java

package com.dlka.lastfmnewtracknotifier;

import com.squareup.moshi.Json;

public class Image {

    @Json(name = "size")
    private String size;
    @Json(name = "#text")
    private String text;

    public String getSize() {
        return size;
    }

    public void setSize(String size) {
        this.size = size;
    }

    public String getText() {
        return text;
    }

    public void setText(String text) {
        this.text = text;
    }

}

File: app/src/main/java/com/dlka/lastfmnewtracknotifier/Image.kt
package com.dlka.lastfmnewtracknotifier

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass


@JsonClass(generateAdapter = true)
data class Image (
    @Json(name = "size")
    var size: String? = null,
    @Json(name = "#text")
    var text: String? = null
)
